ls1intum/Artemis
# Bugfix: resizing the questions and answers component

### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, 

### Motivation and Context


In some cases (especially when the questions and answers component is small) the content of the exercise or lecture overlapped the drag-handle for resizing.

### Description

I added a left margin to the q&a component in order to prevent overlap.

### Steps for Testing


1. Log in to Artemis
(1a. enable questions and answers for your test course if deactivated) 
2. Navigate to an exercise or lecture
3. check that the questions and answers component can be resized along its left border

### Screenshots


make sure this handle can be dragged:
<img width="409" alt="Bildschirmfoto 2020-05-05 um 15 32 54" src="https://user-images.githubusercontent.com/51987021/81071963-bbf0d900-8ee5-11ea-8f05-bac9a0da1936.png">
